package.json

```diff
@@ -38,7 +38,7 @@
     "babel-plugin-transform-regenerator": "^6.26.0",
     "babel-preset-react": "^6.5.0",
     "babel-traverse": "^6.9.0",
-    "babylon": "6.15.0",
+    "babylon": "6.18.0",
```

Comment:
I've upgraded `babylon` and all `babel-*` packages. This was required to get proper support for `opaque` types across the toolchain. All upgrades where minor version only.